jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionDialog.kt

```diff
@@ -235,15 +236,15 @@ class EditFunctionDialog(
 
             ApplicationManager.getApplication().executeOnPooledThread {
                 LambdaFunctionCreator(lambdaClient).update(functionDetails)
-                    .thenAccept { runInEdt(ModalityState.any()) { close(OK_EXIT_CODE) } }
-                    .whenComplete { _, error ->
-                        when (error) {
-                            null -> notifyInfo(
-                                title = NOTIFICATION_TITLE,
-                                content = message("lambda.function.configuration_updated.notification", functionDetails.name)
-                            )
-                            is Exception -> error.notifyError(title = NOTIFICATION_TITLE)
-                        }
+                    .thenAccept {
+                        notifyInfo(
+                            title = NOTIFICATION_TITLE,
+                            content = message("lambda.function.configuration_updated.notification", functionDetails.name)
+                        )
+                        runInEdt(ModalityState.any()) { close(OK_EXIT_CODE) }
```

Comment:
Does this now keep the window open the entire build / packaging process? Do we tell user what is going on or does it just sit there ?

Reply:
No this only fires for config updates